src/main/java/com/kalessil/phpStorm/phpInspectionsEA/utils/BinaryExpressionUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.kalessil.phpStorm.phpInspectionsEA.utils;
+
+import com.intellij.psi.tree.IElementType;
+import com.jetbrains.php.lang.lexer.PhpTokenTypes;
+import com.jetbrains.php.lang.psi.elements.BinaryExpression;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+import org.jetbrains.annotations.NotNull;
+
+public enum BinaryExpressionUtil {
+    ;
```

Comment:
I copied the code from L18~26 from another code, then it was marked as "duplicated". Then I refactored it to make it clean and objective. The file that I have copied is fully compatible with that, but I don't modified to not mix issues, but I give the "door open" to make it possible.